xbmc/platform/darwin/DarwinUtils.mm

```diff
@@ -114,24 +114,19 @@
 const char* CDarwinUtils::getIosPlatformString(void)
 {
   static std::string iOSPlatformString;
-  if (iOSPlatformString.empty())
-  {
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
```

Comment:
Have you thought about using `std::call_once` ?

Reply:
didn't even know about it :) Too used to objc/GCD. Will change.